server/src/test/scala/services/crunch/CrunchSpec.scala

```diff
@@ -30,96 +27,6 @@ class CrunchSpec extends Specification {
     oneDayMillis === 60 * 60 * 24 * 1000
   }
 
-  "Given a start time of zero milliseconds " +
```

Comment:
are these test covered ?

Reply:
Yes the functions were not used in production code so have been removed